### DISCUSSION TITLE
Support for adding nonces for CSP?

### DISCUSSION THREAD

**Discussion by andrewk11** [vega/vega-lite#9502] [Q&A] [answered]
Is there any support/intended support for adding nonces to comply with CSP or using stylesheet-based styles for chart tooltips? I'm working on a project that uses Vega Lite tooltips, and the tooltip inline styling has failed to apply since we required nonces for inline styling.

**Comment by domoritz** [comment, marked as answer]
I think these would be good improvements but I probably won't have time to implement it myself. Happy to review changes, though.

**Reply by andrewk11** [reply]
Got it. I don't think I'll have the bandwidth for that anytime soon, but appreciate the clarity.
